### PR TITLE
Fixing ExamineeAttributes discrepancies between SRP and ERT/TRT output

### PR DESCRIPTION
**Pull request by burzum619**
https://jira.fairwaytech.com/browse/TDS-1061

### CODE REVIEW

File: src/main/java/tds/exam/results/mappers/ExamineeMapper.java [hunk 1]
```diff
@@ -11,6 +19,17 @@
  * A mapper class for mapping examinee data to the JAXB {@link tds.exam.results.trt.TDSReport.Examinee} object
  */
 public class ExamineeMapper {
+    private static final String DEFAULT_BIRTH_DATE = "1900-01-01";
+    private static final String DOB_ATTRIBUTE_ID = "DOB";
+    private static final Logger log = LoggerFactory.getLogger(ExamineeMapper.class);
+
+    private static final ImmutableMap<String, String> attributeNames = ImmutableMap.of(
```
Comment (malaffoon) [inline src/main/java/tds/exam/results/mappers/ExamineeMapper.java]: minor nit: the variable only needs to be Map<String, String>  

In general, i like to use the least specific container possible when declaring variables.

Comment (burzum619) [reply src/main/java/tds/exam/results/mappers/ExamineeMapper.java]: I typically do too, this one just got by me. Thanks for catching that

File: src/main/java/tds/exam/results/mappers/ExamineeMapper.java [hunk 2]
```diff
@@ -27,13 +47,37 @@
...
         });
 
         return examinee;
     }
+
+    /*
+        The birthdate is typically formatted "MMDDYYYY" and is expected to be in "YYYY-MM-DD" - ReportingDLL.readTesteeAttributes()
+     */
+    private static String formatBirthDate(final String dob) {
+        DateFormat tdsFormat = new SimpleDateFormat("MMddyyyy");
```
Comment (millerts) [inline src/main/java/tds/exam/results/mappers/ExamineeMapper.java:73]: Can the dob ever be null?  If so, maybe early return?

**Review by millerts — APPROVED** [review]

File: src/main/java/tds/exam/results/mappers/ExamineeMapper.java (hunk 2 above)
Comment (burzum619) [reply src/main/java/tds/exam/results/mappers/ExamineeMapper.java:73]: No, it will never be null.

**Pull request merged**